README.md

```diff
@@ -19,6 +19,18 @@ Run the image by providing the appropriate volume mount paths and parameters:
     docker run -v "/var/run/docker.sock:/var/run/docker.sock:rw" -v "/root/.ssh:/home/user/.ssh" -v "/tmp:/tmp" -v "/var/tmp/jasminegraph-localstore:/var/tmp/jasminegraph-localstore" -v "/var/tmp/jasminegraph-aggregate:/var/tmp/jasminegraph-aggregate" -v "/home/user/Documents/jasminegraph/metadb:/home/ubuntu/software/jasminegraph/metadb" -v "/home/user/Documents/MSc/jasminegraph/performancedb:/home/ubuntu/software/jasminegraph/performancedb" -p 7777:7777 -p 7778:7778 jasminegraph --MODE 1 --MASTERIP <docker0 interface ip> --WORKERS 4 --WORKERIP <docker0 interface ip> --ENABLE_NMON false
 
 
+Run JasmineGraph in Kubernetes environment by providing appropriate parameters:
+
+    ./start-k8s.sh --META_DB_PATH "$(pwd)/metadb" \
```

Comment:
@Ishad-M-I-M  @thevindu-w  We keep metadb and perfdb folders within the source tree of JasmineGraph project. Once the JasmineGraph executable gets built we keep that also in the same project structure. This is because till now we do not have a concept of installation of JasmineGraph. Therefore, when we run JasmineGraph we keep the metadb and perfdb data on the same source tree. Can you think of an improved approach?

Reply:
The metadb and perfdb are now build from their ddl and the binary db files are excluded from VCS. They now act like temporary files similar to object files (*.o) generated automatically but not version controlled. Since we keep the object files in the same place, I don't see any reason for not keeping the db files there.

Reply:
Ok, I see